administrator/components/com_media/resources/scripts/components/browser/browser.vue

```diff
@@ -115,6 +118,9 @@ export default {
         width: this.$store.state.showInfoBar ? '75%' : '100%',
       };
     },
+    isEmpty() {
+      return ![...this.$store.getters.getSelectedDirectoryDirectories, ...this.$store.getters.getSelectedDirectoryFiles].length;
```

Comment:
```suggestion
      return ![...this.$store.getters.getSelectedDirectoryDirectories, ...this.$store.getters.getSelectedDirectoryFiles].length
      && !this.$store.state.isLoading;
```

The compute `isEmpty` should be aware of the loading state and should not render if the data is not loaded